compiler/test/dotty/tools/vulpix/ParallelTesting.scala

```diff
@@ -504,20 +504,35 @@ trait ParallelTesting extends RunnerOrchestration { self =>
       reporter
     }
 
-    private def parseErrors(errorsText: String, compilerVersion: String) =
-      val errorPattern = """.*Error: (.*\.scala):(\d+):(\d+).*""".r
-      errorsText.linesIterator.toSeq.collect {
-        case errorPattern(filePath, line, column) =>
-          val lineNum = line.toInt
-          val columnNum = column.toInt
-          val abstractFile = AbstractFile.getFile(filePath)
-          val sourceFile = SourceFile(abstractFile, Codec.UTF8)
-          val offset = sourceFile.lineToOffset(lineNum - 1) + columnNum - 1
-          val span = Spans.Span(offset)
-          val sourcePos = SourcePosition(sourceFile, span)
-
-          Diagnostic.Error(s"Compilation of $filePath with Scala $compilerVersion failed at line: $line, column: $column. Full error output:\n\n$errorsText\n", sourcePos)
-      }
+    private def parseErrors(errorsText: String, compilerVersion: String, pageWidth: Int) =
+      val errorPattern = """^.*Error: (.*\.scala):(\d+):(\d+).*""".r
+      val summaryPattern = """\d+ (?:warning|error)s? found""".r
+      val indent = "    "
+      var diagnostics = List.empty[Diagnostic.Error]
+      def barLine(start: Boolean) = s"$indent${if start then "╭" else "╰"}${"┄" * pageWidth}${if start then "╮" else "╯"}\n"
```

Comment:
What version of the compiler and with which options did you use to get his kind of a frame? I've never seen it before

Reply:
It is just [ASCII boxes](https://en.wikipedia.org/wiki/Box-drawing_character). I'm creating the frame here.

Reply:
Yeah, that I know. I was wondering why you're getting this in your compiler output while this doesn't happen to me and I didn't even know we have such a functionality

Reply:
OK, sorry, for some reason I thought it's a part of your usual compiler output and you're trying to parse it here